src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useRef } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+export const Input = ({ onEditComplete, defaultValue = "" }) => {
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
+
+  const handleBlur = () => {
+    onEditComplete(inputRef.current.value);
+  };
+
+  const handleKeyDown = (e) => {
+    if (e.key !== "Enter") return;
+    onEditComplete(inputRef.current.value);
+  };
+
+  return (
+    <StyledInput
+      defaultValue={defaultValue}
+      onBlur={handleBlur}
+      onKeyDown={handleKeyDown}
+      ref={inputRef}
+    />
+  );
+};
+
```

Comment:
このあたり特にいらない気もしますがいかがでしょう？
```suggestion
const StyledInput = styled.input`
```